src/styles/window-top.sass

```diff
@@ -61,3 +62,5 @@
       .title
         font-size: 12px
         color: #a5a2a2
+        overflow: hidden
+        text-overflow: ellipsis
```

Comment:
please also add
```sass
white-space: nowrap
padding: 0 10px
```

then it looks like:
![screen shot 2017-05-15 at 22 17 48](https://cloud.githubusercontent.com/assets/1796022/26077494/85675c28-39bc-11e7-84f3-6e7962c557c2.png)